sphinx_doc/user_guide.rst

```diff
@@ -120,18 +127,20 @@ Manually creating a HMMER database
 
 Creating a HMMER database via command line
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-An admin user can add or remove data from the genomics-workspace database via the command line interface. Here, we describe how to use commands to interact with the database.
+An admin user like i5k can add or remove data from the genomics-workspace database via the command line interface. Here, we describe how to use commands to interact with the database.
 
 1.	To add organism (not necessary if the organism is already added)
 
 * ``python manage.py addorganism [genus] [species]`` (e.g python manage.py addorganism Apis mellifera)
 
-2.	To add hmmer
+2.	To add hmmer database
+
+* ``python manage.py addhmmer [genus] [species] -f [path of fasta file] -d [genus] [species] [annotation name], [sequence type]`` (e.g python manage.py addhmmer Apis mellifera -f  media/blast/db/GCF_003254395.2_Amel_HAv3.1_genomic.fna -d "Apis mellifera Apis_mellifera_Annotation_Release_103, peptides")
```

Comment:
We only support protein databases - Can you change `-f [path of fasta file]` to `-f [path of protein fasta file]`?

Reply:
Done